script/install.sh

```diff
@@ -27,6 +27,12 @@ echo "1.create directory"
 if [ ! -d $installPath ];then
   sudo mkdir -p $installPath
   sudo chown -R $deployUser:$deployUser $installPath
+# If install Path equal to "/" or related path is "/" or is empty, will cause directory "/bin" be overwrite or file adding,
+# so we should check its value. Here use command `realpath` to get the related path, and it will skip if your shell env
+# without command `realpath`.
+elif [[ -z "${installPath// }" || "${installPath// }" == "/" || ( $(command -v realpath) && $(realpath -s "${installPath}") == "/" ) ]]; then
```

Comment:
Is it better moving this logic before 27 line?

Reply:
BTW, what is the usage of```//``` in ```${installPath// }```?

Reply:
> BTW, what is the usage of`//` in `${installPath// }`?

It replaces `/  ` or `  /` into `/`, to avoid users add them as installpath but we can not find it

Reply:
> Is it better moving this logic before 27 line?

will change it later, I am doing some doc changes in my git workspace now